scripts/area-owners-md-to-json.ps1

```diff
@@ -0,0 +1,45 @@
+$owner = "dotnet"
```

Comment:
The intend is that this is only used offline when we're migrating, right?

Reply:
Right. We might reuse the script, but it'll be run manually each time.